src/main/java/com/esri/samples/graphics_update/client_app/MoveGraphicsSample.java

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright 2021 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.esri.samples.graphics_update.client_app;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.*;
+import com.esri.samples.graphics_update.position_sumulator.MessageGenerator;
+import com.esri.samples.graphics_update.position_sumulator.UpdateMessage;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import java.io.File;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+
+public class MoveGraphicsSample extends Application {
+    private MapView mapView;
+    private MessageGenerator messageGenerator;
+    private GraphicsOverlay graphicsOverlay;
+    private HashMap<String, Graphic> vehicles = new HashMap<>();
+
+    public static void main(String[] args) {
+        Application.launch(args);
+    }
+
+    @Override
+    public void start(Stage stage) {
+
+        // set the title and size of the stage and show it
+        stage.setTitle("Vehicle position monitoring app");
+        stage.setWidth(800);
+        stage.setHeight(700);
+        stage.show();
+
+        // create a JavaFX scene with a stack pane as the root node and add it to the scene
+        StackPane stackPane = new StackPane();
+        Scene scene = new Scene(stackPane);
+        stage.setScene(scene);
+
+        // create a MapView to display the map and add it to the stack pane
+        mapView = new MapView();
+        stackPane.getChildren().add(mapView);
+
+        // create an ArcGISMap with a basemap
+        ArcGISMap map = new ArcGISMap(Basemap.createLightGrayCanvasVector());
+
+        // graphics overlay for vehicles
+        graphicsOverlay = new GraphicsOverlay();
+        mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+
+        File stylxFile = new File(System.getProperty("data.dir"), "./resources/Vehicles.stylx");
```

Comment:
you can remove `System.getProperty("data.dir")`, this code is required in the samples for the sample viewer to reference the data directory correctly. It's not needed for a stand alone application

```suggestion
        File stylxFile = new File("./resources/Vehicles.stylx");
```